submitted_models/cerberus_anymal_b_sensor_config_1/launch/spawner.rb

```diff
@@ -0,0 +1,92 @@
+def spawner(_name, _modelURI, _worldName, _x, _y, _z, _roll, _pitch, _yaw)
+  <<-HEREDOC
+  <plugin name="ignition::launch::GazeboFactory"
```

Comment:
Please merge with `master` and change `<plugin>` to `<spawn name='#{_name}'>`

Reply:
https://github.com/osrf/subt/pull/383/commits/dd51902c199ecd0b7e388df7ce40d1b4e59a7709